projects/charger/inc/generic_can_msg.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "can_msg.h"
+#include "status.h"
+
+typedef struct GenericCanMsg {
+  uint32_t id;
+  uint64_t data;
+  size_t dlc;
+  bool extended;
+} GenericCanMsg;
+
+// Converts a GenericCanMsg to a CANMessage.
+StatusCode generic_can_msg_to_can_message(const GenericCanMsg* src, CANMessage* dst);
```

Comment:
Man we really need to check clang-format's rules for pointers

Reply:
We should set `DerivePointerAlignment: false`.

Reply:
See #147. I'll run `make format` before merging so I don't have to manually find all these.